### DISCUSSION TITLE
Creating a new page

### DISCUSSION THREAD

**Discussion by obayesshelton** [lunarphp/lunar#1241] [Q&A]
Hi

I have a table in my database called "Kitchens" for example and I would like to create a page in the admin hub to show this data. Is possible to make a livewire component and that to use the admin hub layout and sit under the `hub` folder e.g `hub/kitchens`

**Comment by glennjacobs** [comment]
Totally. You can add items to the hub's menu, see https://docs.lunarphp.io/admin-hub/extending.html#adding-to-menus

Then it's just a case of using the hub's layout and making your Livewire page.

**Comment by obayesshelton** [comment]
@glennjacobs I don't really understand the docs for this. What exactly is 

```
->name(__('menu.sidebar.tickets'))
->handle('hub.tickets')
->route('')
```

What exactly is the handle? 

The `route` is this the route or the routename that matches the route in my routes file within my laravel project?

**Comment by obayesshelton** [comment]
@glennjacobs are we making `"full-page components"` in LiverWire?

**Comment by obayesshelton** [comment]
Also @glennjacobs how do we use the admin hub layout?

**Comment by obayesshelton** [comment]
Resolved :)

**Comment by omar-bfe** [comment]
@obayesshelton Hello I had the same questions as you how to use the App layout from the admin hub. 
did not figure it out. could you please sjare with us how you created a new page on the admin hub please 
Thanks

**Reply by alecritson** [reply]
To use the app layout, you just need to specify it on your Livewire component:

```php
public function render()
{
    return view('my.view')->layout('adminhub::layouts.app', [
        'title' => 'My Custom Page',
    ]);
}
```
